.release/ci.hcl

```diff
@@ -0,0 +1,96 @@
+schema = "1"
+
+project "vault-csi-provider" {
+  team = "vault"
+  slack {
+    // #vault-releases channel
+    notification_channel = "CRF6FFKEW"
+  }
+  github {
+    organization = "hashicorp"
+    repository = "vault-csi-provider"
+    release_branches = ["master"]
```

Comment:
```suggestion
    release_branches = ["main"]
```

Reply:
Thanks for the catch - surprisingly this was the only mention that needed fixing after the switch.